debezium-core/src/main/java/io/debezium/relational/TableSchemaBuilder.java

```diff
@@ -96,11 +100,14 @@ public TableSchema create(String schemaPrefix, String envelopSchemaName, Table t
         SchemaBuilder valSchemaBuilder = SchemaBuilder.struct().name(schemaNameAdjuster.adjust(schemaNamePrefix + ".Value"));
         SchemaBuilder keySchemaBuilder = SchemaBuilder.struct().name(schemaNameAdjuster.adjust(schemaNamePrefix + ".Key"));
         AtomicBoolean hasPrimaryKey = new AtomicBoolean(false);
+        Set<Column> pkColumns = new HashSet<Column>(table.primaryKeyColumns());
         table.columns().forEach(column -> {
-            if (table.isPrimaryKeyColumn(column.name())) {
+            if (table.isPrimaryKeyColumn(column.name()) 
+                    || (pkMapper != null && pkMapper.matches(tableId.catalog(), tableId.schema(), tableId.table(), column.name()))) {
```

Comment:
Do we need to introduce to not null check? How about provideing default identity mapper?

Reply:
Yes, that would be nice eventually: this could encapsulate the entire business of message key determination. In some cases we already have the fallback to the first unique key; so the overall priority would be:

* use PK columns explicitly given via the new option
* use columns from table PK
* use columns from first unique constraint (some connectors)

Reply:
I have updated the code, introducing a new Key class. But I really do not know how I could find the columns from the first unique constraint.

Reply:
Ok, I was looking for this, it seems this happens right during DDL parsing and the `Table` will have the unique constraint column(s) as PK columns already when you get it here.

So I think you're fine by just doing 

* use PK columns explicitly given via the new option
* use columns from table PK